src/CustomMarshalers/CustomMarshalers.sln

```diff
@@ -0,0 +1,32 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.28307.136
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{E9D942E3-D65A-44F8-835E-831066E41788}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ref", "ref", "{88142883-B4AF-4C3B-9B8F-4501407A824E}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CustomMarshalers", "src\CustomMarshalers.csproj", "{E55E513E-152E-48D1-82B6-DDC9A993D779}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{E55E513E-152E-48D1-82B6-DDC9A993D779}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Do the projects open in VS as expected? Looking at other solution files, I would imagine you want to set this up as:
```text
		{E55E513E-152E-48D1-82B6-DDC9A993D779}.Debug|Any CPU.ActiveCfg = netcoreapp-Windows_NT-Debug|Any CPU
		{E55E513E-152E-48D1-82B6-DDC9A993D779}.Debug|Any CPU.Build.0 = netcoreapp-Windows_NT-Debug|Any CPU
		{E55E513E-152E-48D1-82B6-DDC9A993D779}.Release|Any CPU.ActiveCfg = netcoreapp-Windows_NT-Release|Any CPU
		{E55E513E-152E-48D1-82B6-DDC9A993D779}.Release|Any CPU.Build.0 = netcoreapp-Windows_NT-Release|Any CPU
```

cc @safern